src/error/error_handler.e

```diff
@@ -35,6 +34,11 @@ feature -- Access
 			Result := not error_messages.is_empty
 		end
 
+	error_messages: LIST [ERROR]
```

Comment:
export error_messages ... otherwise TAR_LS does not get compiled.
And it think it makes sense to export this list.